src/averageAges.js

```diff
@@ -7,19 +7,34 @@
  * function calculates average age only for men who died in this century
  *
  * To calculate century:
- * Divide year of person's death by 100: Math.ceil(person.died / 100)
+ * Divide year of personson's death by 100: Math.ceil(personson.died / 100)
  *
  * @param {object[]} people
  * @param {number} century - optional
  *
  * @return {number}
  */
+
+function averageAge(persons) {
+  return persons.map(person => person.died - person.born)
+    .reduce((firstperson, secondperson) =>
+      firstperson + secondperson) / persons.length;
+};
```

Comment:
Checklist point 1: function names should start with a verb. Also it would be more readable if every method in the chain started on a new line. And don't forget about camelCasing your variables
```suggestion
function calculateAverageAge(persons) {
  return persons
    .map(person => person.died - person.born)
    .reduce((firstPerson, secondPerson) =>
      firstPerson + secondPerson) / persons.length;
};
```